src/main/java/bookstore/entity/Category.java

```diff
@@ -13,8 +13,8 @@
 import org.hibernate.annotations.SQLRestriction;
 
 @Entity
-@Table(name = "categories")
-@SQLDelete(sql = "UPDATE category SET is_deleted = 1 WHERE id = ?")
+@Table(name = "category")

```

Comment:
table name should be plural
```suggestion
@Table(name = "categories")
```